optuna/multi_objective/visualization/_pareto_front.py

```diff
@@ -0,0 +1,121 @@
+import json
+from typing import List
+from typing import Optional
+
+from optuna._experimental import experimental
+from optuna.multi_objective.study import MultiObjectiveStudy
+from optuna.multi_objective.trial import FrozenMultiObjectiveTrial
+from optuna.visualization.utils import _check_plotly_availability
+from optuna.visualization.utils import is_available
+
+if is_available():
+    from optuna.visualization.plotly_imports import go
+
+
+@experimental("1.5.0")
```

Comment:
Since we just released 1.5.0.
```suggestion
@experimental("2.0.0")
```

Reply:
🙇